recipes/onnx2torch/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "onnx2torch" %}
+{% set version = "1.5.1" %}
+
+package:
+  name: "{{ name|lower }}"
```

Comment:
No need to quote all the jinja variables.

Reply:
:heavy_check_mark: